foundation-extensions/experience-platform-extension/README.md

```diff
@@ -1,8 +1,8 @@
 # Adobe Experience Platform Edge Network
 
-## Configure the Edge Network extension in Launch
+## Configure the Edge Network extension in Platform
```

Comment:
This page requires more updates:
Change "AEP Edge Network extension" -> "Edge Network extension" in all the sections below